File: src/main/resources/com/example/fxproject/testUtils/python/memoryAllocation.py
import timeit
if __name__ == '__main__':
        # Measure the time taken to access and modify the memory
        start = timeit.default_timer()
        mem = list(range(int(1e4)))
        end = timeit.default_timer()
        # Calculate the duration
        duration = end - start
        print(int(duration * 1000000000 / int(1e4)))

File: src/main/resources/com/example/fxproject/testUtils/memoryAllocation.cpp
#include <iostream>
#include <chrono>
using namespace std;
const int SIZE = 1e8; // Size of the static array
int *mem;
int main() {
    // Measure the time taken to access the static memory
    auto start = std::chrono::high_resolution_clock::now();
    mem = static_cast<int *>(calloc(SIZE, sizeof(int)));
    auto end = std::chrono::high_resolution_clock::now();
    // Calculate the duration
    std::chrono::duration<double> duration = end - start;

    return duration.count() * 100000;
}
